controllers/C_Document.class.php

```diff
@@ -241,7 +241,7 @@ function note_action_process($patient_id) {
         } elseif ($_POST['identifier'] == "yes") {
 
             // send the document via email
-            $d = new Document($_POST['foreign_id']);
+            $d = preg_match("/^\d+$/", $_POST['foreign_id']) ? new Document($_POST['foreign_id']) : new Document();
```

Comment:
I guess this was making sure it's a digit only right.

Reply:
Yep, and it instantiates a Document object accordingly